src/lwc/gsApplicationStatus/gsApplicationStatus.html

```diff
@@ -0,0 +1,37 @@
+<template >
+    <template if:false={isActiveInstance}>
+        <lightning-card>
+            <div class="spinner" if:true={isLoading}>
+                <lightning-spinner alternative-text="Loading"></lightning-spinner>
+            </div>
+            <div class="slds-grid slds-grid_align-center slds-grid_vertical">
```

Comment:
Please use `lightning-layout` and `lightning-layout-item` for layouts, not directly `slds-grid`/`slds-col`. Much easier to read and maintain. https://developer.salesforce.com/docs/component-library/bundle/lightning-layout/example